src/button/button.style.tsx

```diff
@@ -68,26 +76,32 @@ export const Main = styled.button<MainStyleProps>`
                         box-shadow: none;
                     }
 
-                    color: ${Color.Primary};
+                    color: ${isDanger
+                        ? DesignToken.Button.Danger.Primary
+                        : Color.Primary};
                     :hover,
                     :active,
                     :focus {
                         span {
-                            color: ${Color.Secondary};
+                            color: ${isDanger
+                                ? DesignToken.Button.Danger.Hover
+                                : Color.Secondary};
                         }
                     }
                 `;
             default:
                 return css`
-                    background-color: ${Color.Primary(props)};
+                    background-color: ${isDanger
+                        ? DesignToken.Button.Danger.BackgroundColor
+                        : Color.Primary(props)};
                     border: 1px solid transparent;
 
                     ${MediaQuery.MaxWidth.mobileL} {
                         width: 100%;
                     }
 
                     span {
-                        color: ${Color.Neutral[8](props)};
+                        color: ${isDanger ? "white" : Color.Neutral[8](props)};
```

Comment:
Neutral 8 is close to white. Any reason we don't use it?

Reply:
updated to use Neutral 8.

 Was looking for the color , but saw there were some styles that uses "white" . So I went to follow it 

Realize ur right, Neutral 8 is #FFFFFF which is consider white already